web/pandas/community/benchmarks.md

```diff
@@ -0,0 +1,77 @@
+# Benchmarks
+
+Benchmarks are tests to measure the performance of pandas. There are two different
+kinds of benchmarks relevant to pandas:
+
+* Internal pandas benchmarks to measure speed and memory usage over time
+* Community benchmarks comparing the speed or memory usage of different tools at
+  doing the same job
+
+## pandas benchmarks
+
+pandas benchmarks are implemented in the [asv_bench](https://github.com/pandas-dev/pandas/tree/main/asv_bench)
+directory of our repository. The benchmarks are implemented for the
+[airspeed velocity](https://asv.readthedocs.io/en/v0.6.1/) (asv for short) framework.
+
+The benchmarks can be run locally by any pandas developer. This can be done
+with the `asv run` command, and it can be useful to detect if local changes have
+an impact in performance, by running the benchmarks before and after the changes.
```

Comment:
Might link to here? https://pandas.pydata.org/docs/dev/development/contributing_codebase.html#id24

Reply:
Very good point, added.